release_management/_utils.py

```diff
@@ -149,12 +168,13 @@ def create_release(repository, tag, release_version, changelog_url):
     return release
 
 
-def vendor(manifest_abs_path, release_version):
+def vendor(manifest_abs_path, release_version, *, suffix=None):
```

Comment:
Right here, you'll want to take the keyword ```suffix``` argument that you put in back out again. ```release_version```, instead of being a str, should be an actual ```ReleaseVersion``` object. You'll need to document that in the header comments. And you can drop lines 200 - 202 that you previously added. Then, everything _in this method_ should just work, because the f-formatted string evaluation should automatically invoke ```str()```.

But, you'll have to fix up two call sites: one in ```create_artifacts.py``` and one in ```create_release.py```.

Reply:
OK, I just added a new commit, which results in `create_artifacts.py` outputting valid source artifacts, and `generate_sourcerpms.sh` outputs a source RPM that can be successfully built.

Can you review that I made a valid change for `create_release.py`?